### ISSUE TITLE
Update Urdu translations

### ISSUE THREAD

**Issue by nmanumr** [squidfunk/mkdocs-material#4940]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### Language availability

- [X] I've checked the [list of available languages](https://bit.ly/33vFDD0)

### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "ur",
  "action.edit": "اس صفحے میں ترمیم کریں",
  "action.skip": "براہِ راست مواد پر جائیں",
  "action.view": "اس صفحہ کا ماخذ دیکھیں",
  "announce.dismiss": "اسے دوبارہ مت دکھائیں",
  "blog.archive": "محفوظ شدہ",
  "blog.categories": "اقسام",
  "blog.categories.in": "میں",
  "blog.continue": "پڑھنا جاری رکھیے",
  "blog.draft": "ڈرافٹ",
  "blog.index": "واپس انڈیکس پر جائیں",
  "blog.meta": "میٹا ڈیٹا",
  "blog.references": "متعلقہ لنکس",
  "clipboard.copy": "کلِپ بورڈ میں نقل کریں",
  "clipboard.copied": "کلِپ بورڈ میں نقل کر دیا گیا",
  "consent.accept": "قبول کریں",
  "consent.manage": "سیٹینگ بدلیں",
  "consent.reject": "رد کرنا",
  "footer": "ذیلی تحریر",
  "footer.next": "اگلا",
  "footer.previous": "پچھلا",
  "header": "سر تحریر",
  "meta.comments": "تبصرے",
  "meta.source": "ذریعہ",
  "nav": "رہنمائی",
  "readtime.one": "1 منٹ لگے گا",
  "readtime.other": "# منٹ لگیں گے",
  "rss.created": "RSS فیڈ",
  "rss.updated": "تازہ ترین مواد کی RSS فیڈ",
  "search": "تلاش",
  "search.placeholder": "تلاش کریں",
  "search.share": "اشتراک کریں",
  "search.reset": "صاف کریں",
  "search.result.initializer": "تلاش کا آغاز ہو رہا ہے",
  "search.result.placeholder": "تلاش شروع کرنے کے لئے ٹائپ کریں",
  "search.result.none": "کوئی ملتی جلتی دستاویزات نہیں",
  "search.result.one": "۱ ملتی جلتی دستاویز",
  "search.result.other": "# ملتی جلتی دستاویزات",
  "search.result.more.one": "اِس صفحے پر مزید ۱",
  "search.result.more.other": "اِس صفحے پر مزید #",
  "search.result.term.missing": "گمشدہ",
  "select.language": "زبان کا انتخاب کریں",
  "select.version": "ورژن کا انتخاب کریں",
  "source": "ریپازٹری پر جائیں",
  "source.file.contributors": "تعاون کار",
  "source.file.date.created": "تخلیق",
  "source.file.date.updated": "آخری بار تجدید",
  "tabs": "ٹیبز",
  "toc": "فہرست",
  "top": "واپس اوپر جائیں"
}[key] }}{% endmacro %}
```

**Comment by squidfunk** [comment]
Thanks for providing the translations! Added in 664af42367c696abf0d7f58e3903fb44579cde89.

**Comment by squidfunk** [comment]
Released as part of [9.0.8](https://github.com/squidfunk/mkdocs-material/releases/tag/9.0.8).
